src/Administration/Resources/app/administration/src/app/component/media/sw-media-preview-v2/sw-media-preview-v2.spec.js

```diff
@@ -264,4 +264,28 @@ describe('src/app/asyncComponent/media/sw-media-preview-v2', () => {
 
         expect(wrapper.vm.sourceSet).toBe('');
     });
+
+    it('should show warning icon for unsupported video format (MOV)', async () => {
+        const wrapper = await createWrapper();
+        await wrapper.setData({
+            imagePreviewFailed: true,
+            trueSource: { mimeType: 'video/quicktime', thumbnails: [] },
+        });
+        await flushPromises();
+
+        const warningIcon = wrapper.find('.sw-media-preview-v2__warning-icon');
+        expect(warningIcon.exists()).toBe(true);
+    });
+
+    it('should not show warning icon for supported video format (MP4)', async () => {
+        const wrapper = await createWrapper();
+        await wrapper.setData({
+            imagePreviewFailed: true,
+            trueSource: { mimeType: 'video/mp4', thumbnails: [] },
+        });
+        await flushPromises();
+
+        const warningIcon = wrapper.find('.sw-media-preview-v2__warning-icon');
+        expect(warningIcon.exists()).toBe(false);
+    });
```

Comment:
```suggestion
    it.each([
        { mimeType: 'video/quicktime', isSupported: true },
        { mimeType: 'video/mp4', isSupported: false },
    ])(`should show warning icon if video format is not supported (type: ${mimetype}, supported: ${isSupported.toString()})`, async (mimeType, isSupported) => {
        const wrapper = await createWrapper();
        await wrapper.setData({
            imagePreviewFailed: true,
            trueSource: { mimeType, thumbnails: [] },
        });
        await flushPromises();

        const warningIcon = wrapper.find('.sw-media-preview-v2__warning-icon');
        expect(warningIcon.exists()).toBe(isSupported);
    });
```

How about a dataprovider? You can easily add more types to the test and you have a better overview.